### ISSUE TITLE
Streaming with front camera flipped

### ISSUE THREAD

**Issue by wahyu-mediatechindo** [pili-engineering/PLDroidCameraStreaming#83]
How to flip horizontaly video output from front camera?
Right hand person become left hand in video output.

**Comment by jpxiong** [comment]
Did you mean **mirror effect** in front camera ?

**Comment by wahyu-mediatechindo** [comment]
Yes, right. When I streaming with back camera, the video output is normal. But, with front camera video output get flipped because of that. I can't get setting code to do that. It doesn't matter with video preview, but video output is more important.

**Comment by jpxiong** [comment]
Got it. We will provide the related API in the next version.
